### DISCUSSION TITLE
tags folder getting ignored

### DISCUSSION THREAD

**Discussion by StevenXL** [vercel/next.js#12965] [Help] [answered]
Hi folks. 

I have a nextjs project that I am building, and for some reason, `git` is ignoring a directory called `tags`. This is my directory structure:

[Directory Structure](https://ibb.co/rp0tj4q)

If I were to copy the contents of the `tags` folder into a `tests` directory, then `git` will start tracking it.

I haven't modified my `.gitignore` (reproduced in full below). I just can't explain the behavior I am seeing, and I was wondering if the NextJS project somehow treats such a directory as special?

```text
# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.

# dependencies
/node_modules
/.pnp
.pnp.js

# testing
/coverage

# next.js
/.next/
/out/

# production
/build

# misc
.DS_Store

# debug
npm-debug.log*
yarn-debug.log*
yarn-error.log*

# local env files
.env.local
.env.development.local
.env.test.local
.env.production.local
```

Thank you for any help.

**Comment by rafaelalmeidatk** [comment, marked as answer]
You may have another git config somewhere in your system that is ignoring a folder called `tags`. Try to manually add these files with `git add pages/tags/[tag].js`

If the file is being ignored, you will see an output like

> The following paths are ignored by one of your .gitignore files:

If you see this then you have two solutions: find the .gitignore file (you should also look at `.git/info/exclude` and the `core.excludesfile` config) and remove the `tags` folder, or add the files with the `-f` flag: `git add -f pages/tags/[tag].js`

**Reply by StevenXL** [reply]
You are 100% spot on. I used the `-f` flag. Thank you very much!
